src/components/FeedbackButton.tsx

```diff
@@ -0,0 +1,107 @@
+import { EditIcon } from "@chakra-ui/icons";
+import {
+    Modal,
+    ModalOverlay,
+    ModalContent,
+    ModalHeader,
+    ModalFooter,
+    ModalBody,
+    ModalCloseButton,
+    Button,
+    useDisclosure,
+    IconButton,
+    Text,
+    Textarea
+} from "@chakra-ui/react";
+
+import React from "react";
+
+import { submitFeedback } from "../services/FeedbackService";
+
+export function FeedbackButton() {
+    const { isOpen, onOpen, onClose } = useDisclosure();
+    const [feedbackString, setFeedbackString] = React.useState<string>("");
+    const [isFeedbackSubmit, setIsFeedbackSubmit] = React.useState<boolean>(false);
+
+    const handleInputChange = (e: any) => {
+        const inputValue = e.target.value;
+        setFeedbackString(inputValue);
+    };
+
+    const submitForm = async () => {
+        if (feedbackString === "") {
+            alert("Cannot submit empty feedback");
```

Comment:
i'm going to be picky here and say we probably don't want to use alerts because they look ugly. can you have a status text that shows the error instead?